sortedList/sortedList/main.c

```diff
@@ -0,0 +1,100 @@
+#include "list.h"
+
+#include <stdio.h>
+#include <stdbool.h>
+
+void instruction() {
+	printf("\nEnter:\n"
+		"0 to exit\n"
+		"1 to add new value to the list\n"
+		"2 to delete value from the list\n"
+		"3 to print the list\n");
+}
+
+bool test() {
+	struct SortedList* list = createList();
+	if (push(list, 5)) {
+		return false;
+	}
+
+	if (push(list, 2)) {
+		return false;
+	}
+
+	if (!isSorted(list)) {
+		return false;
+	}
+
+	if (!deleteValue(list, 1)) {
+		return false;
+	}
+
+	if (deleteValue(list, 5)) {
+		return false;
+	}
+
+	deleteList(list);
+	return true;
+}
+
+int main() {
+	if (!test()) {
+		printf("There are some problems in the programm :c");
```

Comment:
```suggestion
		printf("There are some problems in the program :c");
```
В английском, в отличие от русского, "программа" пишется с одной "м"